resources/backend.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: backend
+  template:
+    metadata:
+      labels:
+        app: backend
+    spec:
+      containers:
+        - name: backend
+          image: kaktusowy500/backend
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8081
+          env:
+            - name: REDIS_URL
+              value: "hb-redis-micro-a65b2b40-cbbb-450d-aac2-42e2ec61846f-redis.grzyby:6379"
+            - name: REDIS_PASSWORD
```

Comment:
please delete it and mount value from secret:

https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables